packages/plugin-ext/src/plugin/debug/debug-ext.ts

```diff
@@ -262,6 +269,37 @@ export class DebugExtImpl implements DebugExt {
         });
     }
 
+    set activeStackItem(stackItem: theia.DebugStackFrame | theia.DebugThread | undefined) {
```

Comment:
I think we should check whether the active item actually changed before sending notifications.